cli/command/container/opts.go

```diff
@@ -577,6 +579,7 @@ func parse(flags *pflag.FlagSet, copts *containerOptions) (*containerConfig, err
 	hostConfig := &container.HostConfig{
 		Binds:           binds,
 		ContainerIDFile: copts.containerIDFile,
+		ContainerIDEnv:  copts.containerIDEnv,
```

Comment:
Out of curiosity, is this `HostConfig` structure serialized to be passed to the daemon? What format is used for the serialization?

Reply:
sorry I kind of missed this :p

as per my comment [here](https://github.com/balena-os/balena-engine/pull/174#issuecomment-514298726) it's trasmitted as json, the cli uses the same api all other client programs use to talk to the daemon